frontend/src/AlertsManagement/AlertInfo.js

```diff
@@ -82,41 +130,28 @@ class AlertInfo extends Component {
         </center>
         <List className="anomalies-list">
           {alert.anomalies.map((anomaly, index) => {
-            const chartData = {
-              labels: [ ...anomaly.dataPoints.keys() ],
-              datasets: [
-                {
-                  label: anomaly.metricName + ' for ' + anomaly.dimensionName,
-                  fill: false,
-                  lineTension: 0.1,
-                  backgroundColor: 'rgba(75,192,192,0.4)',
-                  borderColor: 'rgba(243, 0, 87, 1)',
-                  borderCapStyle: 'butt',
-                  borderDash: [],
-                  borderDashOffset: 0.0,
-                  borderJoinStyle: 'miter',
-                  pointBorderColor: 'rgba(243, 0, 87, 1)',
-                  pointBackgroundColor: '#fff',
-                  pointBorderWidth: 1,
-                  pointHoverRadius: 5,
-                  pointHoverBackgroundColor: 'rgba(243, 0, 87, 1)',
-                  pointHoverBorderColor: 'rgba(220,220,220,1)',
-                  pointHoverBorderWidth: 2,
-                  pointRadius: 1,
-                  pointHitRadius: 10,
-                  data: [ ...anomaly.dataPoints.values() ]
-                }
-              ]
-            };
-
             return (
               <Fragment key={index}>
                 <ListItem key={index} dense>
-                  <ListItemText id={index} 
-                    primary={`Anomaly in ${anomaly.metricName} for ${anomaly.dimensionName} on ${anomaly.timestampDate}`} 
-                  />
+                  <Grid container>
+                    <Grid item xs={12}>
+                      <ListItemText id={index} 
+                        disableTypography
+                        primary={
+                          <Typography variant="body1" style={{ marginTop: '10px' }}>
+                            {`Anomaly in ${anomaly.metricName} for ${anomaly.dimensionName} on ${anomaly.timestampDate}\n`}
+                          </Typography>
+                        }
+                      />
+                    </Grid>
+                    <Grid item xs={6}>
```

Comment:
does the `6` and the `12` mean how wide the div is going to take and the full length is `12`?

Reply:
yup so 6 is half